packages/runtime/lib/worker/itc.js

```diff
@@ -58,9 +57,11 @@ function setupITC (app, service, dispatcher) {
       await app.listen()
     }
 
-    dispatcher.replaceServer(app.appConfig.useHttp ? app.server.url : app.server)
+    const url = app.stackable.getUrl()
 
-    return service.entrypoint ? app.server.url : null
+    dispatcher.replaceServer(app.appConfig.useHttp ? url : app.server)
```

Comment:
Don't rely on `useHTTP`. Some stackables might never start HTTP.
I would go for:

```
dispatcher.replaceServer(url ?? app.getServer())
```

Reply:
Actually I had to modify this locally. Please use:

```
dispatcher.replaceServer(url ?? app.stackable)
```

Reply:
done